torch/fx/experimental/proxy_tensor.py

```diff
@@ -590,6 +591,35 @@ def unwrap_proxy(self, e):
         else:
             return e
 
+    def create_node(self, *args, **kwargs):
```

Comment:
Maybe we can add to ModuleStackTracer [here](https://fburl.com/code/2hh7kp7a) instead of to the PythonKeyTracer? Just in case normal make_fx users do not care ab having metadata.